src/BitcoinLib/Requests/FundRawTransaction/FundRawTransactionOptions.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections.Generic;
+
+namespace BitcoinLib.Requests.FundRawTransaction
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class FundRawTransactionOptions
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        public Dictionary<string, object> Data = new Dictionary<string, object>();
+
+        /// <summary>
+        /// (string, optional, default=pool address) The Bitcoin Cash address to receive the change
+        /// </summary>
+        public string changeAddress { set => Data[nameof(changeAddress)] = value; }
```

Comment:
This and all the other properties in this file should start with a capital letter, to be consistent with the rest of the request type declarations.